src/hdf5/tropics_2ioda.py

```diff
@@ -344,10 +341,6 @@ def get_WMO_satellite_ID(attrs_shortname):
         WMO_sat_ID = TROPICS06_WMO_sat_ID
     elif 'TROPICS07' in attrs_shortname:
         WMO_sat_ID = TROPICS07_WMO_sat_ID
-    elif 'TMS_S3_BRTTL1B' in attrs_shortname:
-        WMO_sat_ID = TOMORROWIO_S3_sat_ID
-    elif 'TMS_S1_BRTTL1B' in attrs_shortname:
-        WMO_sat_ID = TOMORROWIO_S1_sat_ID
```

Comment:
we will need to add `satelliteSubIdentifier` and I just checked this does not exist in our validation yaml yet... 
https://github.com/JCSDA-internal/ioda/blob/develop/share/ioda/yaml/validation/ObsSpace.yaml